modules/katamari/src/main/ts/ephox/katamari/api/Strings.ts

```diff
@@ -75,3 +77,40 @@ export const lTrim: (s: string) => string =
 
 export const rTrim: (s: string) => string =
   blank(/\s+$/g);
+
+// Extract codepoint a la ES2015 String.fromCodePoint
+// From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint
+export const fromCodePoint = (...codePoints: number[]): string => {
+  if (nativeFromCodePoint) {
+    return nativeFromCodePoint(...codePoints);
+  } else {
+    const codeUnits: number[] = [];
+    let codeLen = 0;
+    let result = '';
+    for (let index = 0, len = codePoints.length; index !== len; ++index) {
+      let codePoint = +codePoints[index];
```

Comment:
Not sure I've ever seen this syntax before so looked it up https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Unary_plus_2

I wonder if we could be using this elsewhere...